### PR TITLE
Webpack host environment variables

### PR DESCRIPTION
**Pull request by sreynen**
The environment variable named `JOB_DB_HOST` was being used to define `webpackHost`, not the actual job database host, so this changes the name to reflect that. Also, `JOB_DB_PORT` wasn't being used at all, while we do have a use for `WEBPACK_PORT`, so changing that as well. I imagine there probably is some potential use for `JOB_DB_` variables, but having them defined before we're actually using them seems to only add potential confusion.

### CODE REVIEW

**Comment by schuyler1d** [comment]
:+1: doubly-good because JOB_DB_PORT wasn't used anywhere!  so confusing

File: webpack/server.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ import webpack from 'webpack'
 import config from './config'
 import { log } from '../src/lib'
 
-const webpackPort = 3000
+const webpackPort = process.env.WEBPACK_PORT
```
Comment (schuyler1d) [inline webpack/server.js]: maybe make this `const webpackPort = process.env.WEBPACK_PORT || 3000`
(and below: `const webpackHost = process.env.WEBPACK_HOST || 'localhost'` )

**Review by shakalee14 — COMMENTED** [review]
looks good to me.

**Comment by shakalee14** [comment]
@sreynen you want to merge this?

**Pull request merged**